master/internal/core_experiment.go

```diff
@@ -366,7 +365,7 @@ func (m *Master) parseCreateExperiment(req *apiv1.CreateExperimentRequest, owner
 		}
 	}
 
-	token, createSessionErr := user.StartSession(ctx, owner)
+	token, createSessionErr := getTaskSessionToken(ctx, owner)
```

Comment:
So, I don't know all the details here but wasn't the issue that the user session was invalid when we were passing it in as a jwt. IIUC, there was a "user token" which we expected to be jwt, and a session token which is made for each experiment. 

Is this change to make sure that the user token that is passed in from the user (the jwt) is actually taken and used  as opposed to making a new one (which is my understanding of the previous functionality).

The names are mostly throwing me off since the name `getTaskSessionToken` suggests to me it is reusing the session token, which from what I understand, would be a much more involved undertaking to get working in lore.

Reply:
>  wasn't the issue that the user session was invalid when we were passing it in as a jwt.

DET_USER_TOKEN in the task env was fine when it came through as a jwt token (ntsc case tested via shell) but was invalid in case of experiments which came back as a local det token

> Is this change to make sure that the user token that is passed in from the user (the jwt) is actually taken and used as opposed to making a new one (which is my understanding of the previous functionality)

yes. it needs some more testing though

Reply:
this now shares the same path as with ntsc https://github.com/determined-ai/determined/blob/d53b1f14699926fb6e1a8b09229b38d7971804b1/master/internal/api_command.go#L145